Erlang support OpenSSL 3 (#277)
asdf-vm/asdf-erlang: README.md

Install the build tools
`brew install autoconf`

Install OpenSSL `brew install openssl`

> Note, Erlang 23 and older, doesn't support openssl 3, [read more here](https://github.com/erlang/otp/issues/4577#issuecomment-925962048): `brew install openssl@1.1`

Reviewer: ```suggestion
Note that Erlang 24.1 and older require OpenSSL 1.1, [read more here](https://github.com/erlang/otp/issues/4577#issuecomment-925962048). In addition, from 24.2 to 25.0 recommend using OpenSSL 1.1 instead of OpenSSL 3.0, especially for production use, [read more here](https://github.com/erlang/otp/releases/tag/OTP-25.1). 
`brew install openssl@1.1`
```

I will add more details about this later.

`brew install autoconf`

Install OpenSSL
`brew install openssl@1.1`  _Erlang 24.1 and older require OpenSSL 1.1, [read more here](https://github.com/erlang/otp/issues/4577#issuecomment-925962048)_

> Note, Erlang 25.1 and newer, [support OpenSSL 3.0, even for production use.](https://github.com/erlang/otp/releases/tag/OTP-25.1) 
> If necessary, you can install the latest version of OpenSSL `brew install openssl`

Reviewer: ```suggestion
Note, Erlang 25.1 and newer [support OpenSSL 3.0, even for production use.](https://github.com/erlang/otp/releases/tag/OTP-25.1) 
If you want to build Erlang with openssl@3.0, install by `brew install openssl`
```

Reviewer: Just a suggestion from my poor English skill though ;D

Reviewer 2: 👍

Author: @takasehideki 
Thanks for good suggestion ✨ 
I applied the suggested change 👍
